gam_app/management/commands/importar_una_bolsa.py

```diff
@@ -235,6 +235,10 @@ def handle(self, *args, **options):
         bolsas_nuevas = bolsas.difference(bolsas_importadas)
         bolsas_nuevas = list(bolsas_nuevas)
 
+        if len(bolsas_nuevas) == 0:
+            print('no hay bolsas nuevas')
+            exit()
```

Comment:
Where is the `exit` function defined?

Reply:
I'll change it to sys.exit() and send a new push.